roles/business.md

```diff
@@ -2,6 +2,22 @@
 
 ## Business Traits
 
+### +Academy
+
+We need someone:
+
+- to prepare and deliver the weekly learning programmes
+- to provide appropriate feedback and coaching, to help academy members to succeed  
+- to use a variety of teaching methods to challenge and engage academy members
+- to be the PO for the greenfield project
+
+We expect someone in the role:
+
+- to be autonomous in leading the academy
```

Comment:
Yeah unsure it's clear what the motive behind this is, unsure it's clear how someone can check to see whether they're being autonomous. Does autonomous mean not asking for help?

Think our expectations should provide clearer guidance.